diagnostic.md

```diff
@@ -3,42 +3,45 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Its an optinionated framework, so a lot of decisions are already made.
+It has ember cli, which has a lot of built in developer tools.
+Ember data can do crud actions easily and interacts well with db\'s
```

Comment:
I think it was looking for the 3 big things mentioned about ember in Fridays lesson which are routability, less AJAX, and responsiveness.